Makefile

```diff
@@ -214,6 +214,7 @@ ifndef DOCKER_PASS
 endif
 
 kind-deploy-controller: check-env
+	kubectl create ns $(KIND_NAMESPACE) --kubeconfig=$(PWD)/kubeconfig_hub
```

Comment:
It seems we also need to create `open-cluster-management-agent-addon` ns on the hub cluster for leader election, correct me if I'm wrong